cse-lib/network-processing/src/main/java/com/farao_community/farao/cse/network_processing/busbar_change/NetworkModifier.java

```diff
@@ -0,0 +1,316 @@
+/*
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.cse.network_processing.busbar_change;
+
+import com.farao_community.farao.commons.FaraoException;
+import com.powsybl.iidm.network.*;
+import com.powsybl.ucte.converter.util.UcteConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This class exposes useful methods to modify a network
+ *
+ * @author Peter Mitri {@literal <peter.mitri@rte-france.com>}
+ */
+public class NetworkModifier {
```

Comment:
I think we should catch PowsyblException somewhere as well and report them properly without the process to crash

Reply:
done in all public methods